tests/testsuite/build_script.rs

```diff
@@ -471,43 +458,31 @@ fn custom_build_invalid_host_config_feature_flag() {
                 target
             ),
         )
-        .file(
-            "build.rs",
-            r#"
-            use std::env;
-
-            fn main() {
-                assert!(env::var("RUSTC_LINKER").unwrap().ends_with("/path/to/linker"));
-            }
-            "#,
-        )
+        .file("build.rs", "fn main() {}")
```

Comment:
Like above, this is being deleted because the build script isn't actually running, right?

Reply:
Exactly.